setup.sh

```diff
@@ -33,9 +33,11 @@ done
 set -- "${POSITIONAL[@]}" # Restore positional parameters
 
 BUILD_ARGS=""
+MVFST_ARGS=""
 if [ "$INFERENCE" = true ]; then
   echo -e "Inference-only build"
   BUILD_ARGS="--inference"
+  MVFST_ARGS="-s"
```

Comment:
This is only relevant when ./setup.sh is called from pantheon right? One could just install inference only mode without going through the pantheon integration. A better way is to have this flag be passed in https://github.com/viswanathgs/pantheon/blob/master/src/wrappers/mvfst_rl.py#L18 and other places instead of hard-coding here.